pkg/sql/opt_catalog.go

```diff
@@ -2429,6 +2429,10 @@ func (oi *optVirtualIndex) InvertedColumn() cat.IndexColumn {
 
 // Predicate is part of the cat.Index interface.
 func (oi *optVirtualIndex) Predicate() (string, bool) {
+	if oi.idx != nil {
```

Comment:
nit: putting the `if oid.idx == nil { ... }` case first would match the other methods and be easier to read IMO

Reply:
Done.